packages/core/src/components/Form.tsx

```diff
@@ -1,34 +1,34 @@
-import React, { Component } from "react";
 import {
```

Comment:
import React, { Component } from "react";
```suggestion
import {
```